### PR TITLE
fix: fetch error when opening eth send

### LINKED ISSUE #2536: Opening Send ETH modal is causing error in console
**Describe the bug**
Opening Send ETH modal is causing error in console. Also I can see many other errors that may slow the wallet.

Image
Image

**To Reproduce**
1. Go to ETH account
2. Click on Send ETH modal
3. Open console

**Expected behavior**
Should act without errors 

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Chrome, FF
 - Version 119

**Additional context**
Reproducible on the dev ETH branch: https://feat-eth-support.wallet.z52da5wt.xyz/account

### PR DESCRIPTION
**Pull request by martinkaintas**
closes #2536

### CODE REVIEW

File: src/popup/components/TransferSend/TransferSendRecipient.vue [hunk 1]
```diff
@@ -85,9 +85,10 @@ export default defineComponent({
   setup(props) {
     const { openModal } = useModals();
     const { activeAccount } = useAccounts();
-    const { getTippingUrlStatus } = useAeTippingUrls();
+    const isAeternity = computed(() => props.protocol === PROTOCOL_AETERNITY);
```
Comment (peronczyk) [inline src/popup/components/TransferSend/TransferSendRecipient.vue:88]: It's more common to name such variables `isAe` within the repo.

File: src/popup/components/TransferSend/TransferSendRecipient.vue [hunk 2]
```diff
@@ -85,9 +85,10 @@ export default defineComponent({
...
+    const { getTippingUrlStatus } = useAeTippingUrls(isAeternity.value);
 
-    const urlStatus = computed(() => getTippingUrlStatus(props.modelValue));
+    const urlStatus = computed(() => isAeternity.value ? getTippingUrlStatus(props.modelValue) : 'not-verified');
```
Comment (peronczyk) [inline src/popup/components/TransferSend/TransferSendRecipient.vue:91]: If you are using string values please force the return type to validate it:
```suggestion
    const urlStatus = computed((): UrlStatus => isAeternity.value ? getTippingUrlStatus(props.modelValue) : 'not-verified');
```

File: src/protocols/aeternity/composables/aeTippingUrls.ts [hunk 3]
```diff
@@ -11,7 +11,7 @@ function getTwitterAccountUrl(url: string) {
 const verifiedUrls = ref<string[]>([]);
 const blacklistedUrls = ref<string[]>([]);
 
-export function useAeTippingUrls() {
+export function useAeTippingUrls(ensureFetchedOnInit = true) {
```
Comment (peronczyk) [inline src/protocols/aeternity/composables/aeTippingUrls.ts:14]: It's more common to pass the composable options as an object to make it easy to extend and modify it.

File: src/popup/components/TransferSend/TransferSendRecipient.vue (hunk 2 above)
Comment (martinkaintas) [reply src/popup/components/TransferSend/TransferSendRecipient.vue]: Absolutely correct, I wasn't aware this is possible with computed properties

File: src/popup/components/TransferSend/TransferSendRecipient.vue [hunk 4]
```diff
@@ -85,7 +85,8 @@ export default defineComponent({
   setup(props) {
     const { openModal } = useModals();
     const { activeAccount } = useAccounts();
-    const { getTippingUrlStatus } = useAeTippingUrls();
+    const isAe = computed(() => props.protocol === PROTOCOL_AETERNITY);
+    const { getTippingUrlStatus } = useAeTippingUrls({ ensureFetchedOnInit: isAe.value });
```
Comment (peronczyk) [inline src/popup/components/TransferSend/TransferSendRecipient.vue:89]: Please group the composables initialization and move the `isAe` computed further down to group it with other computed props.

**Review by peronczyk — APPROVED** [review]

**Comment by Liubov-crypto** [comment]
Fixed, LGTM

**Pull request merged**